moveit_ros/benchmarks/include/moveit/benchmarks/BenchmarkExecutor.h

```diff
@@ -170,8 +170,7 @@ class BenchmarkExecutor
   void shiftConstraintsByOffset(moveit_msgs::msg::Constraints& constraints, const std::vector<double>& offset);
 
   /// Check that the desired planner plugins and algorithms exist for the given group
-  bool plannerConfigurationsExist(const std::map<std::string, std::vector<std::string>>& planners,
-                                  const std::string& group_name);
+  bool plannerConfigurationsExist(const std::map<std::string, std::vector<std::string>>& planners);
```

Comment:
Why did we take off the group name? And either way, the docstring then needs to be updated?